### PR TITLE
13080 New schema for BEN IA and special resolution Collab Correction Schema Updates for BEN IA

### PR DESCRIPTION
**Pull request by lewischenstudio**
*Issue #:* /bcgov/entity#13080
*Issue #:* /bcgov/entity#13502

*Description of changes:*
- added fields in example data for CORRECTION_INCORPORATION
- added fields for correction.json

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-schemas license (Apache 2.0).

### CODE REVIEW

File: src/registry_schemas/schemas/special_resolution.json [hunk 1]
```diff
@@ -32,6 +32,17 @@
...
                     "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/person"
+                },
+                "resolutionConfirmed": {
+                    "type": "boolean",
+                    "title": "Confirmation of resolution"
+                },
+                "resolutionText": {
+                    "type": "string",
+                    "maxLength": 1000
+                },
```
Comment (argush3) [inline src/registry_schemas/schemas/special_resolution.json:43]: `resolutionText` already exists as `resolution`.

File: src/registry_schemas/schemas/special_resolution.json [hunk 2]
```diff
@@ -32,6 +32,17 @@
...
+                "signingPerson": {
+                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/person"
                 }
```
Comment (argush3) [inline src/registry_schemas/schemas/special_resolution.json:46]: `signingPerson` already exists as `signatory`

File: src/registry_schemas/schemas/special_resolution.json (hunk 1 above)
Comment (lewischenstudio) [reply src/registry_schemas/schemas/special_resolution.json:43]: Sure. I'll tell Faiz to change "resolutionText" to "resolution".

File: src/registry_schemas/schemas/special_resolution.json (hunk 2 above)
Comment (lewischenstudio) [reply src/registry_schemas/schemas/special_resolution.json:46]: Sure. I'll tell Faiz to change "signingPerson" to "signatory".

File: src/registry_schemas/schemas/special_resolution.json [hunk 3]
```diff
@@ -32,6 +32,17 @@
                 },
                 "signatory": {
                     "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/person"
+                },
+                "resolutionConfirmed": {
+                    "type": "boolean",
+                    "title": "Confirmation of resolution"
+                },
```
[the remark this answers is not in the document]
Comment (lewischenstudio) [reply src/registry_schemas/schemas/special_resolution.json:39]: I got those from the recent special resolution interface. I also see that colin uses this too. I think it should be okay. I'll let Faiz know about this.

[the remark this answers is not in the document]
Comment (lewischenstudio) [reply src/registry_schemas/schemas/special_resolution.json]: I've asked Faiz about this. Let's wait for his response.

[the remark this answers is not in the document]
Comment (argush3) [reply src/registry_schemas/schemas/special_resolution.json]: I added a note to the PR to clarify my thoughts on it too.

[the remark this answers is not in the document]
Comment (lewischenstudio) [reply src/registry_schemas/schemas/special_resolution.json]: I just found out this property isn't used any other place than state-models.ts for initialization. I added a question to verify with Faiz.

[the remark this answers is not in the document]
Comment (lewischenstudio) [reply src/registry_schemas/schemas/special_resolution.json]: "resolutionConfirmed" is removed from that PR. If this looks good to you, I think I want to wait to merge that PR before merging this one.

[the remark this answers is not in the document]
Comment (argush3) [reply src/registry_schemas/schemas/special_resolution.json]: sounds good.  just a note that you may need to update the version number if other changes get merged before yours.

**Review by argush3 — APPROVED** [review]

**Review by vysakh-menon-aot — APPROVED** [review]

File: src/registry_schemas/schemas/correction.json [hunk 4]
```diff
@@ -116,6 +116,68 @@
...
+                "enum": [
+                  "BEN"
+                ]
+              }
+            }
+          },
+          "then": {
+            "required": [
+              "nameRequest",
+              "offices",
```
Comment (lekshmimallika-aot) [inline src/registry_schemas/schemas/correction.json:134]: I don't think 'nameRequest' and 'offices' should be required properties.  If you are not correcting name request then you need not pass that object at all. Same with offices. Party can be mandatory if the completing party is always mandatory

**Review by lekshmimallika-aot — APPROVED** [review]

**Pull request merged**